lib/ui/window.dart

```diff
@@ -5,8 +5,8 @@ part of dart.ui;
 
 /// A view into which a Flutter [Scene] is drawn.
 ///
-/// Each [FlutterView] has its own layer tree that is rendered into an area
-/// inside of a [FlutterWindow] whenever [render] is called with a [Scene].
+/// Each [FlutterView] has its own layer tree that is shown whenever [render]
+/// is called on it with a [Scene].
```

Comment:
Sorry for being pedantic, but there's no guarantee a rendered scene is shown. What do you think of something like this?

```suggestion
/// Each [FlutterView] has its own layer tree that is rendered 
/// whenever [render] is called on it with a [Scene].
```